beacon-chain/sync/pending_blocks_queue.go

```diff
@@ -160,6 +161,11 @@ func (s *Service) processPendingBlocks(ctx context.Context) error {
 			default:
 			}
 
+			if err := s.requestPendingBlobs(ctx, b.Block(), blkRoot[:]); err != nil {
```

Comment:
So i actually think this might be the wrong place for this: 
we request missing parent roots here below:
```
s.sendBatchRootRequest(ctx, parentRoots, randGen)
```
if we also want the corresponding blobs for these blocks we should be requesting them together. First block then its blob